src/api-reference/invoice/purchase-request-endpoints.markdown

```diff
@@ -3,50 +3,32 @@ title: Purchase Request v4
-
-In addition a Get Status endpoint is provided that can be used to get basic information and check the status of the created purchase request. 
-
-> **Limitations**: This API does not support users in the China data center.
-
-* [Products and Editions](#products-editions)
-* [Scope Usage](#scope-usage)
-* [Dependencies](#dependencies)
-* [Access Token Usage](#access-token-usage)
 * [Create a new purchase request](#post)
 * [Get details of purchase request](#get)
 * [Schema](#create_purchase_request_schema)
-* [Response schema](#create_purchase_request_schema-response)
+  * [Create Purchase Request Schema](#create_purchase_request_schema)
+  * [Create Purchase Response Schema](#create_purchase_request-response-schema)
+  * [Get Purchase Request Response Schema](#get_purchase_request_response_schema)
+  * [Error](#error)
 * [Error codes](#error-codes)
 
-## <a name="products-editions"></a>Products and Editions
-
-* Concur Invoice Professional Edition
-* Concur Invoice Standard Edition
-
-## <a name="scope-usage"></a>Scope Usage
-
-Name|Description|Endpoint
----|---|---
-`PurchaseRequest.Read`|Read only access to purchase request data|Get details of a purchase request
-`PurchaseRequest.Write`|Read and Write access to purchase request data|Create a new purchase request
-
-## <a name="dependencies"></a>Dependencies
-
-None
-
-## <a name="access-token-usage"></a>Access Token Usage
-
-This API will work with both Company or User access tokens, however a Company `access token` is recommended for integrations using this API if the end goal is for the integration to create purchase requests for multiple requestors. When using a User `access token` to create purchase requests through the API it will result in the purchase request being assigned to the user that generated the User `access token` and it will not honor the user set in the payload. A User `access token` could be used for testing to check if your payloads are good if needed. For an integration as mentioned above a Company `access token` is the best choice.   
+## Version
+4.0<br>
 
 ## <a name="post"></a>Create a new purchase request  
 
     POST  /purchaserequest/v4/purchaserequests
 
 Create a Purchase Request based on provided header and line item details. If the request is valid it returns back a unique identifier to look at purchase request details and creates a purchase request in back ground.
 
+### <a name="8"></a>Scopes 
+
+|Name | Description
+|-----|------------
+|purchaserequest.write|Create a new purchase request
+
 ### <a name="8"></a>Parameters  
 
 |Name | Type | Format | Description
```

Comment:
A few changes in the Parameters table- For Authorization, since we do allow user access tokens for this API, let's make the final sentence "This is the company or user access token"
purchaseRequest is not a parameter, it's the Payload so it shouldn't be in this table.

Reply:
Fixed in 771852fa